.env.example

```diff
@@ -1,17 +1,22 @@
-MONGODB_USER=root
-MONGODB_PASSWORD=123456
-MONGODB_DATABASE=appnet_dev
-MONGODB_LOCAL_PORT=7017
-MONGODB_DOCKER_PORT=27017
+DB_HOST=localhost
+DB_USER=root
+DB_PASSWORD=123456
+DB_NAME=identity_network_dev
+DB_PORT=7017
+DB_DOCKER_PORT=27017
+DB_LOCAL_PORT=7017
 NODE_LOCAL_PORT=8000
 NODE_DOCKER_PORT=8000
 
 HEDERA_NETWORK=testnet
-HEDERA_OPERATOR_ID="0.0.xxxx"
-HEDERA_OPERATOR_KEY="302xxxxx"
+HEDERA_OPERATOR_ID="0.0.28520500"
```

Comment:
@vijayshiyani not sure if it is intentional but I would suggest to put `0.0.xxxxxxxx` to not confuse users